examples/main.tf

```diff
@@ -218,12 +218,6 @@ resource "propel_metric" "custom_metric" {
   dimensions = ["column_1", "column_2"]
 }
 
-resource "propel_policy" "sum_metric_policy" {
```

Comment:
there is a `access_control_enabled` in the metric, lets review if that as well needs to be deprecated